src/Modules/Backend/assets/installer/translations.xml

```diff
@@ -1589,6 +1674,10 @@
         <translation locale="nl"><![CDATA[Het ingegeven wachtwoord is te kort.]]></translation>
         <translation locale="en"><![CDATA[The password is too short.]]></translation>
       </item>
+      <item type="error" name="EmailIsTooLong">
```

Comment:
Where is this used? And why is it needed?

Reply:
In the user DTO, has to do with max length of the field in the database because of indexes. Otherwise it would get cut off without letting the user know but on login the email address wouldn't match. Email max length is set at 180

Reply:
Ok, good to know.
I don't know if this is something that should be documented in the `User`-entity class. Because according to the RFC about email addresses:

> That limit is a maximum of 64 characters (octets) in the "local part" (before the "@") and a maximum of 255 characters (octets) in the domain part (after the "@") for a total length of 320 characters. However, there is a restriction in RFC 2821 on the length of an address in MAIL and RCPT commands of 254 characters.

Maybe just add a comment to explain that it is because of the indexes in the database.